django_auth_ldap/__init__.py

```diff
@@ -1,2 +1,2 @@
-VERSION = (2, 2, 0)
+VERSION = (2, 2, 1)
```

Comment:
Shouldn’t be needed for this PR.
```suggestion
VERSION = (2, 2, 0)
```

Reply:
This has been removed.